spec/controllers/idv/cancellations_controller_spec.rb

```diff
@@ -188,6 +190,41 @@
         expect(response).not_to render_template(:destroy)
         expect(parsed_body).to eq({ redirect_url: account_path })
       end
+
+      context 'with in person enrollment' do
+        let(:user) { build(:user, :with_pending_in_person_enrollment) }
+
+        before do
+          allow(IdentityConfig.store).to receive(:in_person_proofing_enabled).and_return(true)
+          allow(controller).to receive(:user_session).and_return(
+            'idv/in_person' => { 'pii_from_user' => {},
+                                 'Idv::Steps::Ipp::StateIdStep' => true,
+                                 'Idv::Steps::Ipp::AddressStep' => true },
+          )
+        end
+
+        it 'cancels pending in person enrollment' do
+          expect(user.reload.pending_in_person_enrollment).to_not be_blank
+          delete :destroy
+
+          expect(user.reload.pending_in_person_enrollment).to be_blank
+        end
+
+        it 'cancels establishing in person enrollment' do
+          create(:in_person_enrollment, :establishing, user: user)
+          expect(InPersonEnrollment.where(user: user, status: :establishing).count).to eq(1)
+          delete :destroy
+
+          expect(InPersonEnrollment.where(user: user, status: :establishing).count).to eq(0)
+        end
```

Comment:
I recommend being a bit stricter for these tests, i.e. verify that these specific enrollments are in a "cancelled" state, in addition to verifying that the user has no other pending/establishing enrollments.

Reply:
Good call, updated in 4312bd866